packages/block-library/src/navigation/index.php

```diff
@@ -469,27 +469,25 @@ function render_block_core_navigation( $attributes, $content, $block ) {
 
 		// Only published posts are valid. If this is changed then a corresponding change
 		// must also be implemented in `use-navigation-menu.js`.
-		if ( 'publish' !== $navigation_post->post_status ) {
-			return '';
-		}
+		if ( 'publish' === $navigation_post->post_status ) {
+			$nav_menu_name = $navigation_post->post_title;
 
-		$nav_menu_name = $navigation_post->post_title;
-
-		if ( isset( $seen_menu_names[ $nav_menu_name ] ) ) {
-			++$seen_menu_names[ $nav_menu_name ];
-		} else {
-			$seen_menu_names[ $nav_menu_name ] = 1;
-		}
+			if ( isset( $seen_menu_names[ $nav_menu_name ] ) ) {
+				++$seen_menu_names[ $nav_menu_name ];
+			} else {
+				$seen_menu_names[ $nav_menu_name ] = 1;
+			}
 
-		$parsed_blocks = parse_blocks( $navigation_post->post_content );
+			$parsed_blocks = parse_blocks( $navigation_post->post_content );
 
-		// 'parse_blocks' includes a null block with '\n\n' as the content when
-		// it encounters whitespace. This code strips it.
-		$compacted_blocks = block_core_navigation_filter_out_empty_blocks( $parsed_blocks );
+			// 'parse_blocks' includes a null block with '\n\n' as the content when
+			// it encounters whitespace. This code strips it.
+			$compacted_blocks = block_core_navigation_filter_out_empty_blocks( $parsed_blocks );
```

Comment:
In line 474 of the old one, we close the conditional. Here now we are doing the next commands inside that conditional, I will check 🤔

Reply:
It's not a linting issue, is because the change [in this commit](https://github.com/WordPress/gutenberg/pull/43580/commits/82c732dcfe4138ef9d531fbc1e3a9594629a330b) added a conditional to execute this part, causing an indentation requirement.